tests/processor/connector/test_snapshot_aws.py

```diff
@@ -190,8 +190,7 @@ def test_get_node():
     assert val is not None
     val = get_node(awsclient, {
         "snapshotId": "10",
-        "callable_method" : "get_bucket_acl",
-        "type": "regions",
+        "type": "get_bucket_acl",
```

Comment:
U have replaced regions with get_bucket_acl
Can we not have a separate testcase to get_bucket_acl